message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -133,7 +133,7 @@
       </entry>
       <entry value="42426" name="MAV_CMD_DO_CANCEL_MAG_CAL" hasLocation="false" isDestination="false">
         <description>Cancel a running magnetometer calibration.</description>
-        <param index="1" label="Bitmask" minValue="0" maxValue="255" increment="1">Bitmask of magnetometers to cancel a running calibration (0 means all).</param>
+        <param index="1" label="Magnetometers Bitmask" minValue="0" maxValue="255" increment="1">Bitmask of magnetometers to cancel a running calibration (0 means all).</param>
```

Comment:
@WickedShell Should "0 means all" change/does it have same semantics?

Assuming so:

> Bitmask of magnetometers for which calibration is to be cancelled. Use 0 to cancel calibration for all sensors on which calibration is running. The command will NACK if a sensor is explicitly specified on which calibration is is not running.